packages/esm-generic-patient-widgets-app/src/resources/useObs.ts

```diff
@@ -4,7 +4,9 @@ import { openmrsFetch, fhirBaseUrl, useConfig, FHIRResource, FHIRCode } from '@o
 export const pageSize = 100;
 
 export function useObs(patientUuid: string): UseObsResult {
-  const { data } = useConfig();
+  const { encounterTypes, data } = useConfig();
+
+  const UrlEncounterTypes: string = encounterTypes.length ? `&encounter.type=${encounterTypes.toString()}` : '';
```

Comment:
```suggestion
  const urlEncounterTypes: string = encounterTypes.length ? `&encounter.type=${encounterTypes.toString()}` : '';
```

Reply:
Seems better to name this variable `encounterTypes`.

Reply:
I used the suggested change 'urlEncounterTypes'